imperfect-perfect.tex

```diff
@@ -757,29 +770,35 @@ \section{Iwasawa cohomology and the cyclotomic deformation}
 \begin{equation} \label{eq:psi sequence}
 0 \to M^{\varphi=1} \to M^{\psi=1} \stackrel{\varphi-1}{\longrightarrow} M^{\psi=0}.
 \end{equation}
-The pairing $\{-, - \}$ satisfies
-\begin{equation} \label{eq:pairing phi psi}
-\{\varphi(x), y\} = \{x, \psi(y)\} \qquad (x \in M, y \in M^*).
-\end{equation}
 \end{defn}
 
 \begin{prop} \label{P:psi finite}
 For any $M \in \PhiGamma_{K,A}$,
 the $A$-module $M/(\psi-1)$ is finitely generated.
 \end{prop}
 \begin{proof}
-Apply \cite[Proposition~3.3.2(1)]{kpx}.
+It suffices to treat the case $K = F$.
+The case $F = \QQ_p$ is treated in \cite[Proposition~3.3.2(1)]{kpx}; the same argument applies in general.
```

Comment:
Instead of "in general", which seems contrary to assuming K=F, maybe "applies to general F".